keyboards/ymdk/sp64/rules.mk

```diff
@@ -10,16 +10,16 @@ BOOTLOADER = bootloadhid
 BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
 MOUSEKEY_ENABLE = no        # Mouse keys
 EXTRAKEY_ENABLE = yes       # Audio control and System control
-CONSOLE_ENABLE = no        # Console for debug
+CONSOLE_ENABLE = no         # Console for debug
 COMMAND_ENABLE = yes        # Commands for debug and configuration
 NKRO_ENABLE = no            # Enable N-Key Rollover
-#BACKLIGHT_ENABLE = yes       # Enable keyboard backlight functionality
-AUDIO_ENABLE = no    # There is no speaker on this PCB
+# BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
+AUDIO_ENABLE = no           # There is no speaker on this PCB
```

Comment:
```suggestion
AUDIO_ENABLE = no           # Audio output
```

Reply:
I do not see audio output for the PCB in question -- shouldn't there be an explicit warning in the config that this option will do nothing?

Reply:
Not really, no need.  Setting it to "no" indicates that.  You can add `AUDIO_SUPPORTED = no` to ensure that.

Mostly, it's part of the default block of rules, that we want to keep consistent.

Reply:
Done.